source/Calamari.AzureWebApp/Util/AzureWebAppHelper.cs

```diff
@@ -31,5 +36,36 @@ public static AzureTargetSite GetAzureTargetSite(string siteAndMaybeSlotName, st
             targetSite.Slot = slotName;
             return targetSite;
         }
+       
+        public static async Task<bool> GetBasicPublishingCredentialsPoliciesAsync(string subscriptionId, string resourceGroupName, string webAppName, string accessToken)
+        {
+            var url = $"https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{webAppName}/basicPublishingCredentialsPolicies/scm?api-version=2021-01-15";
```

Comment:
I think this could use the AccountVariables.ResourceManagementEndpointBaseUri for the `https://management.azure.com` section.